cmd/podman/containers/create.go

```diff
@@ -145,6 +145,21 @@ func createInit(c *cobra.Command) error {
 	if c.Flag("no-hosts").Changed && c.Flag("add-host").Changed {
 		return errors.Errorf("--no-hosts and --add-host cannot be set together")
 	}
+	if c.Flag("userns").Changed {
```

Comment:
Can't you handle the shm-size like this, then we can go back to revealing to the user the system default, using the standard mechanism.

Reply:
good idea.  Fixed it